Machine.h

```diff
@@ -69,6 +69,8 @@ typedef struct Machine_ {
    uid_t maxUserId;  /* recently observed */
    uid_t userId;  /* selected row user ID */
 
+   pid_t maxProcessId; /* encountered */
```

Comment:
What does "encountered" in this line comment mean?

Reply:
I think, wording like this is a bit clearer:

```suggestion
   pid_t maxProcessId; /* largest PID seens at runtime */
```

Reply:
Isn't "seens" a typo?